install/prereqs/index.rst

```diff
@@ -31,6 +31,15 @@ System prerequisites
 
   **New since 14.0**: The minimum :command:`cmake` version required to compile the Stack is **2.8.12**.
 
+.. _filesystem-prereqs:
+
+Filesystem prerequisites
+========================
+
+The filesystems on which the stack is built and on which outputs from stack execution are stored must support the `flock` system call for file locking.
```

Comment:
The double "on which" is a bit of an earful to me. What do you think of this edit?

```
Filesystems used for compiling the Stack and hosting output data repositories must support the ``flock`` system call for file locking.
```

Note that the `flock` needs double back ticks in reStructuredText.

Finally, can we give users any hints as to whether their filesystem supports `flock`?